### ISSUE TITLE
Getting search term

### ISSUE THREAD

**Issue by ZorGleH** [select2/select2#247]
Hi, is there a way to get the current term in the search input?
I tried `$('.select2-search input').val()` but the input field is cleared before I can catch it's content when I click somewhere else on the page...

Thanks.

**Comment by mrnoway** [comment]
one year passed: Anyone solved this?

I am currently looking for the same stuff.

**Comment by kevin-brown** [comment]
Without knowing the use case, it's going to be impossible ti find a solution to this question.

**Comment by mrnoway** [comment]
I took a long look at at the compiled results and this gave me a proper solution.

setInterval(function(){
        console.log( $(".select2-sizer").html() );        
    },1000);

The focused input box (actually div) has always an unique class "select2-sizer".
The above code gives me every second the so far written search input.

(better ) I guess if you add an ng-model into the template you could use a watch function to trigger every change without interval.

Why I need this?
Basically to load asynchronous stuff. I did not understand the provided ajax example, so I wanted to build a one on my own :)

**Comment by rongrat9** [comment]
jd
